lutierigb/aws-servicebroker
# adding 5.7.23 to the mapping

the default engine version is set to 5.7.23 but it's not in the Version2Family mapping so CFN refuses to create the stack



## Overview

Brief description of what this PR does, and why it is needed (use case)?
The default engine version in the rdsmysql template is currently set to 5.7.23 but that version is not in the Version2Family mapping, causing the stack to fail validation and it won't even be created. This results in the service broker failing to provision the serviceInstance

## Related Issues

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #116

## Testing
Deployed the stack manually in CFN

### Notes


## Testing Instructions
Deploy a ServiceInstance for rdsmysql not specifying any engineversion, should default to 5.7.23

 How to test this PR Start after checking out this branch (bulleted)
 * Include test case, and expected output

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
